src/Sentry.Extensions.Logging/SentryLogger.cs

```diff
@@ -130,12 +129,24 @@ public bool IsEnabled(LogLevel logLevel)
                    // often used by users experimenting with Sentry
                    && !CategoryName.StartsWith("Sentry.")
                    && !string.Equals(CategoryName, "Sentry", StringComparison.Ordinal)
-                   && (_options.Filters == null
-                        || _options.Filters.All(
-                           f => !f.Filter(
-                               CategoryName,
-                               logLevel,
-                               eventId,
-                               exception)));
+                   && _options.Filters?.All(
+                       f => !f.Filter(
+                           CategoryName,
+                           logLevel,
+                           eventId,
+                           exception)) != false;
+
+        private bool ShouldAddBreadcrumb(
+            LogLevel logLevel,
+            EventId eventId,
+            Exception exception)
+            => _options.MinimumBreadcrumbLevel != LogLevel.None
+               && logLevel >= _options.MinimumBreadcrumbLevel
```

Comment:
`m`: Can we please add a test that checks for the `MinimumBreadcrumbLevel`. Same goes for `MinimumEventLevel` in `ShouldSendEvent`.